src/getHumanAge.test.js

```diff
@@ -7,4 +7,48 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('getHumanAge(0, 0) === [0, 0]', () => {
```

Comment:
Imagine that the developer needs to write code based on your tests. I'm sure you will spend extra time explaining the meaning to your poor colleague. So change all summaries in your test

Reply:
make them more informative

Reply:
Pardon me. Something went wrong when I have been editing the file..